Adapter/PhpcrOdmAdapter.php

```diff
@@ -190,12 +214,54 @@ public function getReferringAutoRoutes($contentDocument)
     public function findRouteForUri($uri, UriContext $uriContext)
     {
         $path = $this->getPathFromUri($uri);
+        $document = $this->dm->find(null, $path);
```

Comment:
I think this could equally work with `->dm->find('AutoRouteInterface', $path)` ?